addons/tmf_loadouts/loadouts/us_rangers_2010.hpp

```diff
@@ -176,40 +174,46 @@ class sl : ftl {
 		"ItemCompass",
 		"ItemWatch",
 		"ACE_VectorDay",
-		"ItemGPS",
 		"ItemRadio"
 	};
 };
 
 class co : sl {
 	displayName = "Platoon Leader";
-	headgear[] = {"H_MilCap_mcamo"};
 };
 
 class fac : co {
 	displayName = "Forward Air Controller";
-	backpack[] = {"CUP_B_AssaultPack_ACU"};
+	goggles[] = {"antenna_long_two_face"};
+	backpack[] = {"B_Kitbag_rgr"};
 	backpackItems[] = { FAC_GEAR };
+	magazines[] += {
+		LIST_2("CUP_1Rnd_SmokeRed_M203"),
+		LIST_2("CUP_1Rnd_SmokeGreen_M203"),
+		"Laserbatteries"
+	};
 	linkedItems[] = {
 		"ItemMap",
 		"ItemCompass",
 		"ItemWatch",
 		"CUP_SOFLAM",
-		"ItemGPS",
 		"ItemRadio"
 	};
 };
 
 class ar : r {
 	displayName = "Automatic Rifleman";
-	primaryWeapon[] = {"hlc_lmg_mk46"};
-	attachment[] = {
-		"cup_acc_anpeq_15_flashlight_tan_l",
-		"cup_acc_anpeq_15"
+	vest[] = {"CUP_V_PMC_CIRAS_OD_TL"};
+	primaryWeapon[] = {"hlc_m249_pip4"};

```

Comment:
His the M249 in the same mod as the Mk48/46?

Reply:
Yeah, I think they're all in the M249 pack... I think. Updated anyways